### PR TITLE
Initial setup of Rails 5.2 app

### PR DESCRIPTION
**Pull request by davidboll**
#### Description
Initial setup of Rails 5.2 app with testing and coverage
Changes proposed in this pull request:
* Adds RSpec
* Adds Cucumber
* Adds Haml
* Adds Travis
* Adds Coveralls
#### What I have learned working on this feature: 
How to Scaffold a Rails 5.2 app

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+ruby '2.4.1'
+
+gem 'haml-rails'
+gem 'rails', '~> 5.2.0.rc1'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
```
Comment (tochman) [inline Gemfile]: I would remove this

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.5'
```
Comment (tochman) [inline Gemfile]: I would remove this

**Review by tochman — COMMENTED** [review]
Please see comments

File: Gemfile (hunk 2 above)
Comment (AKidd95) [reply Gemfile]: Thanks, will do

File: coverage/.last_run.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "covered_percent": 0.0
+  }
+}
```
Comment (tochman) [inline coverage/.last_run.json]: Please remove coverage files

**Review by tochman — CHANGES_REQUESTED** [review]

File: coverage/.last_run.json (hunk 3 above)
Comment (davidboll) [reply coverage/.last_run.json]: Your wish is my command :)

**Review by tochman — APPROVED** [review]

**Comment by tochman** [comment]
Looks good to me now. I would translate the application layout templates to HAML but that can be done in another PR. This one is ready to merge IMO

**Pull request merged**